src/python/doc/cubical_complex_ref.rst

```diff
@@ -11,3 +11,15 @@ Cubical complex reference manual
    :show-inheritance:
 
    .. automethod:: gudhi.CubicalComplex.__init__
+
+.. autoclass:: gudhi.PeriodicCubicalComplex
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+   .. automethod:: gudhi.PeriodicCubicalComplex.__init__
+
+.. autoclass:: gudhi.CubicalComplexBaseClass
```

Comment:
I didn't know this feature, and it seems clearer to me. Done on e1e27ad3
Tell me if you want me to rollback the 2 documentation pages (one for cubical and one for periodic cubical), as I merged the 2 pages because of the base class.